lib/IPC/Run/Win32Helper.pm

```diff
@@ -444,8 +444,15 @@ sub win32_spawn {
 
    my $process;
    my $cmd_line = join " ", map {
-      ( my $s = $_ ) =~ s/"/"""/g;
-      $s = qq{"$s"} if /[\"\s]|^$/;
+      my $s = $_;
+      if ($s eq "") {
+         $s = '""';
+      } elsif ($s =~ /[\s"]/) {
+         # Double-quotes will be escaped with a backslash, and any backslash
+         # that immediately precedes a double-quote will also be escaped.
+         $s =~ s/(\\*)"/$1$1\\"/g;
```

Comment:
\\* would match on nothing. so doing this recursively seems like it'd be a problem.